equistore-torch/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Torch seems to require features added in CMake 3.15, but does not
+# declare it in its own `cmake_minimum_required`
+cmake_minimum_required(VERSION 3.15)
+
+if(NOT "${LAST_CMAKE_VERSION}" VERSION_EQUAL ${CMAKE_VERSION})
+    set(LAST_CMAKE_VERSION ${CMAKE_VERSION} CACHE INTERNAL "Last version of cmake used to configure")
+    if (${CMAKE_CURRENT_SOURCE_DIR} STREQUAL ${CMAKE_SOURCE_DIR})
+        message(STATUS "Running CMake version ${CMAKE_VERSION}")
+    endif()
+endif()
+
+project(equistore_torch CXX)
+
+option(EQUISTORE_TORCH_TESTS "Build equistore-torch C++ tests" OFF)
+
+# Set a default build type if none was specified
+if (${CMAKE_CURRENT_SOURCE_DIR} STREQUAL ${CMAKE_SOURCE_DIR})
+    if("${CMAKE_BUILD_TYPE}" STREQUAL "" AND "${CMAKE_CONFIGURATION_TYPES}" STREQUAL "")
+        message(STATUS "Setting build type to 'release' as none was specified.")
+        set(
+            CMAKE_BUILD_TYPE "release"
+            CACHE STRING
+            "Choose the type of build, options are: none(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) debug release relwithdebinfo minsizerel."
+            FORCE
+        )
+        set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS release debug relwithdebinfo minsizerel none)
+    endif()
+endif()
+
+find_package(equistore 0.1 CONFIG REQUIRED)
+
+# FindCUDNN.cmake distributed with PyTorch is a bit broken, so we have a
+# fixed version in `cmake/FindCUDNN.cmake`
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH}")
+
+find_package(Torch 1.11 REQUIRED)
```

Comment:
here you require 1.11 and in the toml you require 1.13

Reply:
👍

Reply:
Done in #233